### PR TITLE
doc: update product reference name  as AppStudio

### PR DESCRIPTION
**Pull request by happybhati**
Update the product name to `RHTAP` and remove `HACBS` reference.

### CODE REVIEW

File: book/enterprise-contract.md [hunk 1]
```diff
@@ -132,7 +132,6 @@ Additional Resources
 --------------------
 
 - [Enterprise Contract Documentation](https://enterprise-contract.github.io/)
-- [HACBS Documentation](https://red-hat-hybrid-application-cloud-build-services-documentation.pages.redhat.com/hacbs-documentation/)
```
Comment (arewm) [inline book/enterprise-contract.md:135]: Is there a replacement for this URL?

**Review by arewm — APPROVED** [review]
Thanks for the catch!

File: book/enterprise-contract.md (hunk 1 above)
Comment (happybhati) [reply book/enterprise-contract.md:135]: @arewm I did a quick look but didn't find anything.

Comment (happybhati) [reply book/enterprise-contract.md:135]: Found this Archived repo https://gitlab.cee.redhat.com/red-hat-trusted-application-pipeline/stonesoup-documentation

Comment (arewm) [reply book/enterprise-contract.md:135]: There is https://redhat-appstudio.github.io/docs.appstudio.io ... but I don't know if that fits the same content. I don't know what used to be at the previous link.

Comment (scoheb) [reply book/enterprise-contract.md:135]: best to ask the EC team?

@lcarva @simonbaird ?

Comment (lcarva) [reply book/enterprise-contract.md:135]: I suppose this should link to the RHTAP Documentation. But isn't that what this repo is? Maybe it's redundant to have it listed on this page.

**Review by arewm — CHANGES_REQUESTED** [review]
I don't know why I didn't catch this before. `s/RHTAP/AppStudio/g`. We are not using RHTAP in the book.

**Review by arewm — APPROVED** [review]

**Review by lcarva — APPROVED** [review]

**Review by scoheb — APPROVED** [review]

**Pull request merged**